src/Explorer/Panes/SettingsPane/index.tsx

```diff
@@ -0,0 +1,257 @@
+import { Checkbox, ChoiceGroup, IChoiceGroupOption, SpinButton } from "office-ui-fabric-react";
+import React, { FunctionComponent, MouseEvent, useState } from "react";
+import * as Constants from "../../../Common/Constants";
+import { configContext } from "../../../ConfigContext";
+import { LocalStorageUtility, StorageKey } from "../../../Shared/StorageUtility";
+import * as StringUtility from "../../../Shared/StringUtility";
+import { userContext } from "../../../UserContext";
+import { logConsoleInfo } from "../../../Utils/NotificationConsoleUtils";
+import Explorer from "../../Explorer";
+import { GenericRightPaneComponent, GenericRightPaneProps } from "../GenericRightPaneComponent";
+import { Tooltip } from "./Tooltip";
+
+export interface SettingsPaneProps {
+  explorer: Explorer;
+  closePanel: () => void;
+}
+
+export const SettingsPane: FunctionComponent<SettingsPaneProps> = ({
```

Comment:
It would be nice to add a screen capture in the PR, such as in #579.

Reply:
added